packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -106,7 +106,7 @@ const Dashboard = observer(() => {
     React.useEffect(() => {
         if (active_tab === BOT_BUILDER) {
             if (is_drawer_open) {
-                initTrashCan(400);
+                initTrashCan(400, -748);
```

Comment:
any particular reason why we are putting `-748`? @rupato-deriv 
If we really need this I'd suggest to create a constant variable in constants file and use it.

Reply:
because blockly needed this value to calculate the position of the trash can..